backend/test/Test/DashboardSpec.hs

```diff
@@ -80,34 +80,36 @@ spec = withContext $ do
               , nserChanges = one (SqlId changedMeasurement)
               }
         let
-          step :: Word32 -> Measurement -> WithId 'MeasurementId MeasurementResp
-          step i m = WithId (SqlId i) MeasurementResp
-            { mrConcentration = mConcentration m
-            , mrSignal = mSignal m
-            , mrIsEnabled = (i == changedMeasurement) == mIsOutlier m
+          step :: Word32 -> Measurement -> WithId 'MeasurementId Measurement
+          step i m = WithId (SqlId i) Measurement
+            { mConcentration = mConcentration m
+            , mSignal = mSignal m
+            , mIsOutlier = (i == changedMeasurement) == mIsOutlier m
```

Comment:
1. Why did you change `MeasurementResp` to `Measurement`? Is it still needed?
2. Changing `mIsOutlier` value looks like a hack here in any case. `mIsOutlier` tells us whether this measurement was marked as outlier in source data. Once we read it, we never change it. Unlike `mrIsEnabled` which may differ in different sub-experiments.

Reply:
It was needed when 4pl computing took `Measurement` as param